commit 4f9f41c2114ed7f409a7ede40c1f78b1bc8777ba (refs/changes/73/373/1)
Author: David Pacheco <dap@joyent.com>
Date:   2016-08-29T20:15:02+00:00 (3 years, 1 month ago)
    
    The node-fast2 repository on GitHub has been renamed to node-fast.  To avoid dealing with rename in Gerrit, I'm going to reimport "node-fast" and decommission "node-fast2".  To decomission "node-fast2", I'm changing its parent project from GitHub-Replication so that it will be no longer replicated.
    
    Change-Id: Ic07eef7f1d65add114bd922a2c6d729131de5dc6

diff --git a/project.config b/project.config
index 941ee0c..6624721 100644
--- a/project.config
+++ b/project.config
@@ -1,7 +1,7 @@
 [project]
 	description = Authoritative source for http://github.com/joyent/node-fast2
 [access]
-	inheritFrom = GitHub-Joyent
+	inheritFrom = All-Projects
 [submit]
 	action = fast forward only
 	mergeContent = false
